static/js/components/Source.jsx

```diff
@@ -80,154 +58,18 @@ const Source = ({ route }) => {
-          <Link to={`/upload_photometry/${source.id}`} role="link">
-            <Button variant="contained">Upload additional photometry</Button>
-          </Link>
-          <Link to={`/share_data/${source.id}`} role="link">
-            <Button variant="contained">Share data</Button>
-          </Link>
-        </Responsive>
-        <Responsive
-          element={FoldBox}
-          title="Spectroscopy"
-          mobileProps={{ folded: true }}
-        >
-          <Plot
-            className={styles.plot}
-            url={`/api/internal/plot/spectroscopy/${source.id}`}
-          />
-          <Link to={`/share_data/${source.id}`} role="link">
-            <Button variant="contained">Share data</Button>
-          </Link>
-        </Responsive>
-        {/* TODO 1) check for dead links; 2) simplify link formatting if possible */}
-        <Responsive
-          element={FoldBox}
-          title="Surveys"
-          mobileProps={{ folded: true }}
-        >
-          <SurveyLinkList id={source.id} ra={source.ra} dec={source.dec} />
-        </Responsive>
-        <Responsive
-          element={FoldBox}
-          title="Follow-up"
-          mobileProps={{ folded: true }}
-        >
-          <FollowupRequestForm
-            obj_id={source.id}
-            instrumentList={instrumentList}
-            instrumentFormParams={instrumentFormParams}
-          />
-          <FollowupRequestLists
-            followupRequests={source.followup_requests}
-            instrumentList={instrumentList}
-            instrumentFormParams={instrumentFormParams}
-          />
-          <AssignmentForm
-            obj_id={source.id}
-            observingRunList={observingRunList}
-          />
-          <AssignmentList assignments={source.assignments} />
-        </Responsive>
-      </div>
-
-      <div className={styles.rightColumn}>
-        <Responsive
-          element={FoldBox}
-          title="Comments"
-          mobileProps={{ folded: true }}
-          className={styles.comments}
-        >
-          <CommentList />
-        </Responsive>
+  if (width <= 1200) {
+    return (
+      <Paper ref={ref} elevation={1}>
+        <SourceMobile source={source} />
+      </Paper>
+    );
+  }
 
-        <Responsive
-          element={FoldBox}
-          title="Classifications"
-          mobileProps={{ folded: true }}
-          className={styles.classifications}
-        >
-          <ClassificationList />
-          <ClassificationForm
-            obj_id={source.id}
-            action="createNew"
-            taxonomyList={taxonomyList}
-          />
-        </Responsive>
-        <Responsive
-          element={FoldBox}
-          title="Centroid Plot"
-          mobileProps={{ folded: true }}
-        >
-          <Suspense fallback={<div>Loading centroid plot...</div>}>
-            <CentroidPlot className={styles.smallPlot} sourceId={source.id} />
-          </Suspense>
-        </Responsive>
-      </div>
-    </div>
+  return (
```

Comment:
Any reason not to combine these into a single return statement with something like 
``` javascript
{width <= 1200 ? <SourceMobile source={source} /> : <SourceDesktop source={source} />}
```
?